web/content/js/nav.js

```diff
@@ -67,15 +58,57 @@ let navItems = [
 		href: "/Application_Centric_Deployment/index-foreman-el.html"
 	}]
 },{
-	title: "Katello documentation", items: [
+	title: "Foreman on Debian", items: [
+	{
+		title: "Release Notes",
+		href: "/Release_notes/index-foreman-deb.html"
+	},{
+		title: "Quickstart Guide",
+		href: "/Quickstart_Guide/index-foreman-deb.html"
+	},{
+		title: "Installation Guide",
+		href: "/Installing_Server_on_Red_Hat/index-foreman-deb.html"
+	},{
+		title: "Installing Smart Proxy",
+		href: "/Installing_Proxy_on_Red_Hat/index-foreman-deb.html"
+	},{
+		title: "Upgrading and Updating",
+		href: "/Upgrading_and_Updating/index-foreman-deb.html"
```

Comment:
Needs a complete overhaul before this is usable. 100% RPM focused and a lot of Katello specifics.

Reply:
Agreed.